visualization/app/codeCharta/util/customConfigBuilder.ts

```diff
@@ -107,8 +107,11 @@ function initializeDynamicSettings(target: CustomConfig) {
 		sortingOption: undefined,
 		colorRange: {
 			from: 0,
-			to: 0
```

Comment:
Side note for @BridgeAR: Do you remember the decision how to implement the CustomConfigBuilder? Nice to see that the properties are not missed here.